File: Semana1(ori)/app/src/main/java/com/example/semana1/MainActivity.java
package com.example.semana1;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.ImageView;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private TextView loginTitle;
    private ImageView loginLogo;
    private EditText loginUserName;
    private EditText loginPassword;
    private Button loginIniciarBtn;

    //Primer metodo que se ejecuta en una actividad
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        //Referenciar
        loginTitle = findViewById(R.id.loginTitle);
        loginLogo = findViewById(R.id.loginLogo);
        loginUserName = findViewById(R.id.loginUserName);
        loginPassword = findViewById(R.id.loginPassword);
        loginIniciarBtn = findViewById(R.id.loginIniciarBtn);

    }

    //Control + o


}

File: MyApplication2/app/src/main/java/edu/co/icesi/myapplication/MainActivity.kt
package edu.co.icesi.myapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class MainActivity : AppCompatActivity() {


    private lateinit var alfa: Button
    private lateinit var beta: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        alfa = findViewById(R.id.alfa)
        beta = findViewById(R.id.beta)

        alfa.setOnClickListener {
            beta.text = "Funciona!"
        }
    }
}